src/styles/claudy_menu.css

```diff
@@ -0,0 +1,158 @@
+[role=banner] .coz-claudy .coz-claudy-menu {
+  position: fixed;
+  bottom: 9.5em;
+  right: 2em;
+  width: 25em;
+  border-radius: .3em;
+  background: white;
+  transform-origin: 100% 100% 0;
+  transform: scale(0) translateY(4em);
+  filter: drop-shadow(0 4px 6px rgba(50, 54, 63, 0.5));
+  opacity: 0;
+  transition: .2s transform ease-in, .1s opacity ease-in;
+}
+
+[role=banner] .coz-claudy--opened .coz-claudy-menu {
+  transform: scale(1) translateY(0);
+  opacity: 1;
+  transition: .2s transform cubic-bezier(0.2, 0.75, 0.3, 1.15), .1s opacity ease-in;
+}
+
+@media (min-width: 64em) {
+  [role=banner] .coz-claudy .coz-claudy-menu {
+    bottom: 6.5em;
+  }
+}
+
+[role=banner] .coz-claudy-menu .coz-claudy-menu-header {
+  display: flex;
+  flex-direction: row;
+  justify-content: space-between;
+  height: 2em;
+  padding: .75em;
+  background: var(--pale-grey);
+  border-radius: .3em .3em 0 0;
+  border-bottom: 1px solid var(--silver);
+  overflow: hidden;
+}
+
+[role=banner] .coz-claudy-menu .coz-claudy-menu-title {
+  opacity: 1;
+  transition: .2s all ease-in-out;
+}
+
+[role=banner] .coz-claudy-menu .coz-claudy-menu-header-back-button {
+  position: absolute;
+  width: 2em;
+  height: 2em;
+  background: url('../assets/icons/24/icon-arrow-left.svg') center no-repeat;
```

Comment:
Thanks, I was searching quickly for this shorthand but I didn't find with the background size 👍